pom.xml

```diff
@@ -134,6 +135,10 @@ under the License.
                 <artifactId>cassandra-all</artifactId>
                 <version>${cassandraVersion}</version>
                 <exclusions>
+                    <exclusion>
```

Comment:
jna exclusion needs to be removed as dependency is explicitly specified in dependency management section. that will help us to keep jna version consistent(via enforcer plugin) in case of the next C* upgrades.

Reply:
updated